src/screens/home/Home.tsx

```diff
@@ -55,6 +55,22 @@ const Home = ({ navigation, route }: NativeStackScreenProps<HomeStackParams>) =>
     const [interestOptionsModal, setInterestOptionsModal] = useState<boolean>(false);
     const [savedInterestLoading, setSavedInterestLoading] = useState<boolean>(false);
 
+    const fetchUserData = async () => {
```

Comment:
This function exists in 5 different files and is exactly the same. We may possibly want to consider moving it to one file at some point (Maybe somewhere in `firebaseUtils.ts` or `./src/helper`)

Reply:
[beafe77](https://github.com/TAMUSHPE/MobileApp/pull/490/commits/beafe7719457d540a140c7b62143a738b8abbd66)

Reply:
When implementing this, the useContext method isn't allowed in .ts file, so  It seem like a few line of code so i didn't bother too much. 
But extracting it out anyways does make it look better.

Reply:
> When implementing this, the useContext method isn't allowed in .ts file, so It seem like a few line of code so i didn't bother too much. But extracting it out anyways does make it look better.

Oh, I see. Fair enough.